evaluate.sh

```diff
@@ -3,9 +3,9 @@
 . config
 . lib.sh
 
-parse_args "$0" "experiment model model_owner eval_set=eval eval_version=None" "$@"
-shift $n
 check_config "S3_BUCKET OWNER IMAGE PROJECT"
+parse_args "$0" "experiment model model_owner=${OWNER} eval_set=eval eval_version=None" "$@"
+shift $n
```

Comment:
We should do the same for paraphrase.sh right? To choose the filtering model owner. Or perhaps it's time to adopt the "owner/name" convention everywhere.